src/client.toit

```diff
@@ -660,6 +667,11 @@ class Client:
     if connection_:
       connection_.close
       connection_ = null
+      remove_finalizer this
+
+  finalize_:
```

Comment:
We don't do this kind of printing in general. Maybe we should support something like this, but it feels like there is a great risk that you end up using a library that causes this kind of printing and there is no way of getting rid of it. I'd drop this for now and revisit later.